### DISCUSSION TITLE
Frequency of calibration if charge limiter used but not plugged in constantly? Usually: 80%-40% then recharged

### DISCUSSION THREAD

**Discussion by kadbar** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1249] [Q&A]
I use the charge limiter feature regularly, which usually means using my battery 80%-40% then recharging.
I was wondering if this also needs calibration every other week or it is different from the 80% constant charge scenario?
Thanks.

**Comment by MatthiasKerbl** [comment]
Hi @kadbar,

Unfortunately, we haven't looked into your specific use case in detail. However, it should be enough if you use the Calibration Mode about once a month.

Best,
Matthias
